### PR TITLE
Bugfix/hydrator 163

### PR DESCRIPTION
**Pull request by rohitcmu**
https://issues.cask.co/browse/CDAP-8403
https://issues.cask.co/browse/HYDRATOR-163

### CODE REVIEW

**Comment by tonybach** [comment]
Adding the labels created unnecessary space under the pipeline description. This is different from the mocks.

screen shot 2017-03-13 at 1 27 30 pm

Also, we're still showing the tooltips when hovering over the icons. These should be removed.

screen shot 2017-03-13 at 1 30 05 pm

**Comment by leacd** [comment]
For the "schedule" button I want to propose the following changes:
- Clock icon changes in color to reflect states, no other elements are added (e.g. no "pause" icon)
- Labels are descriptives: 
• Start Schedule
• Suspend Schedule 
- Labels are organized on two lines.

**Comment by ajainarayanan** [comment]
Not sure if the label + icons are aligned vertically. 
screen shot 2017-03-14 at 12 04 54 pm

File: cdap-ui/app/directives/widget-container/widget-factory.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ angular.module(PKG.name + '.commons')
           'ng-trim': 'false',
           'ng-model': 'model',
           'rows': '{{myconfig["widget-attributes"].rows}}',
-          placeholder: '{{::myconfig["widget-attributes"].default}}'
+          placeholder: '{{::myconfig["widget-attributes"].placeholder}}'
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/widget-container/widget-factory.js:45]: We should have this backward compatible. If there are user plugins that has widget jsons using textarea or text widgets with default widget attributes, then those might suddenly not show any placeholders.

```
...
placeholder: '{{::myconfig["widget-attributes].placeholder || ::myconfig["widget-attributes"].default }}'
...
```

Comment (ajainarayanan) [reply cdap-ui/app/directives/widget-container/widget-factory.js:45]: Talked to @elfenheart. Right now we use `default` to have a default value and when the default value is set we no longer show the placeholder.

**Comment by ajainarayanan** [comment]
@elfenheart Just noticed the spinner icon we show in the schedule seems to be aligned to the top. 
schedule-spinning

Do we want to align that to center?

**Comment by ajainarayanan** [comment]
we need to fix the width for `.run-specific-section` in the top panel to include the width of the resource center button. Lower browser widths (950px) it seems to be overlapping on top of delete.
screen shot 2017-03-14 at 6 14 43 pm

**Comment by leacd** [comment]
LGTM

**Comment by ajainarayanan** [comment]
👍 LGTM

**Comment by ajainarayanan** [comment]
release/4.1 changes are merged in this PR: https://github.com/caskdata/cdap/pull/8354

**Pull request merged**